_data/changelogs/component-summary-box.yml

```diff
@@ -5,6 +5,11 @@ changelogURL:
 # Add a list of changelog items with newest items first. Uncomment the 'items' data field when adding the first item.
 
 items:
+  - date: NNNN-NN-NN
+    summary: Added WCAG compliance tag and accessibility test status section.
+    affectsGuidance: true
+    githubPr: 2903
```

Comment:
**issue:** This should point to this PR

```suggestion
    githubPr: 3016
```

Reply:
Resolved in [924bdc1](https://github.com/uswds/uswds-site/pull/3016/commits/924bdc18c76c5cc710c5ee47d88bc13761ff66e4)